packages/perspective-viewer-d3fc/src/js/series/categoryPointSeries.js

```diff
@@ -0,0 +1,46 @@
+/******************************************************************************
+ *
+ * Copyright (c) 2017, the Perspective Authors.
+ *
+ * This file is part of the Perspective library, distributed under the terms of
+ * the Apache License 2.0.  The full license can be found in the LICENSE file.
+ *
+ */
+import * as d3 from "d3";
+import * as fc from "d3fc";
+import {tooltip} from "../tooltip/tooltip";
+import {withoutOpacity} from "./seriesColours";
+
+const symbols = [d3.symbolCircle, d3.symbolCross, d3.symbolDiamond, d3.symbolSquare, d3.symbolStar, d3.symbolTriangle, d3.symbolWye];
```

Comment:
Perhaps we could extract the common `fromDomain` into a "symbolScale" module (along with the `symbols` constant it uses)?